xml/Microsoft.Build.BuildEngine/Import.xml

```diff
@@ -80,6 +85,11 @@
         <AssemblyVersion>3.5.0.0</AssemblyVersion>
         <AssemblyVersion>4.0.0.0</AssemblyVersion>
       </AssemblyInfo>
+      <Attributes>
```

Comment:
I don't see this attribute on MSDN https://msdn.microsoft.com/en-us/library/microsoft.build.buildengine.import.isimported(v=vs.100).aspx

wondering if we didn't use to check attributes for the getters and setters before

Reply:
It seems like we have not documented the attribute on MSDN and are now. cc: @joelmartinez

Reply:
I think I'll just go with filtering those out and see what the results are.